src/OrbitGl/TimeGraph.cpp

```diff
@@ -64,6 +64,7 @@ TimeGraph::TimeGraph(AccessibleInterfaceProvider* parent, OrbitApp* app,
   text_renderer_static_.SetViewport(viewport);
   batcher_.SetPickingManager(picking_manager);
   track_manager_ = std::make_unique<TrackManager>(this, viewport_, &GetLayout(), app, capture_data);
+  track_manager_->GetOrCreateSchedulerTrack();
```

Comment:
Agree. I first wanted to move into a direction where the `TrackManager` provides a more generic `AddTrack()` method and does not create any tracks itself. Then its only responsibility would be managing tracks that are created elsewhere. But this would have been to big of a change considering I really only wanted to add a few lines of code to allow hiding by type, so I decided to take a smaller step.